internal/impl/core/metrics.go

```diff
@@ -105,8 +108,9 @@ type Metrics interface {
 
 // Implementation
 type ccsmpBackedMetrics struct {
-	session *ccsmp.SolClientSession
-	metrics []uint64
+	session       *ccsmp.SolClientSession
```

Comment:
Which indentation is our coding standard, before or after?

Reply:
Go has its own standard for formatting for running `go fmt` on the project.
Not sure indentation is included in that for all cases.
If go fmt is happy will this the indentation standard for solace by default is 4 spaces. Unless there is overriding factor like `go fmt`.

Reply:
@gszol, I just run the usual `go fmt` on the code before/while committing the code changes.

Reply:
If everyone's happy, I'm happy :-)